contracts/crowdloan/ICrowdloan.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.5.11;
+
+contract ICrowdloan {
+    function amountRepaid () external returns (uint256);
+    function borrower () external returns (uint256);
+    function crowdfundStart () external returns (uint256);
```

Comment:
@adibas03 I think we should remove the Crowdloan completely

* Ines "mints" 60,000 ERC20 
* "Sells" them on Uniswap (i.e. this replaces the Crowdloan)

As such, we can just focus our efforts 100% on the repayment router

Reply:
I see.
So, simplest implementation possible